modules/eventing/pages/eventing-rbac.adoc

```diff
@@ -0,0 +1,141 @@
+
+== Privileged Users
+
+If a user role of either "Full Admin" or "Eventing Full Admin" it by default has all the necessary access to every resource in a cluster required to run the Eventing Service and create and manage Eventing Functions.
+
+When creating and Eventing Function either of these roles can set the *Function Scope* to *+*+.+*+* no other RBAC role is allowed to use this *Function Scope*.
+
+NOTE: When upgrading to 7.1 all Eventing Functions are assumed to be running as a privileged user and have their *Function Scope* set to *+*+.+*+* to ensure continuity of your Eventing Functions.
+
+=== Full Admin v. Eventing Full Admin
+
+Prior to 7.0.0 Eventing always run as "Full Admin" this blocked some use cases and adoption as this role allowed creation of new users and the ability to escalate privilege sets. The  "Eventing Full Admin" role introduced in 7.0 simply removes the capability of creating users and assigning/modifying RBAC credentials thus providing a bit more security.
+
+For the Function Scope or RBAC grouping we will use the 'bulk.data' assuming you have the role of either "Full Admin" or "Eventing Full Admin". For standard or non-privileged users refer to Role-Based Access Control.
+
+== Eventing and RBAC for Non-privileged Users
+
+NOTE: If a user role of either "Full Admin" or "Eventing Full Admin" it by default has all the necessary access to every resource in a cluster required to run the Eventing Service and create and manage Eventing Functions.
+
+_In RBAC although you can assign rolls directly to a *USER* it is generally more flexible if you define a *GROUP* and assign that group or set of roles to a *USER*, this allows reusing a *GROUP* across multiple users._
+
+== Minimal Eventing RBAC role
+
+The following minimal resources are required for a non-privileged user to access the Eventing Service and create and manage Eventing Functions.
+
+* Data Reader
+** Eventing Storage keyspace or Scratchpad
+* Data Writer
+** Eventing Storage keyspace or Scratchpad
+* Data DCP reader
+** Mutation Source
+* Eventing / Manage Scope Function
+** bucket.scope or bucket.*
+
+== Minimal Eventing RBAC role example
+
+* Access the Couchbase Web Console > Security and Select "ADD GROUP".
++
+image::rbac_min_add_add_group.png[,%100]
+
+* Configure the group as follows "Data Reader" and "Data Writer" are required for the Eventing Storage or scratchpad. "Data DCP Reader" is required to fetch the mutations form DCP. Note this item was defined as 'bulk.data.*' which would allow building Evening functions that can listen to any collection under 'bulk.data'.
++
+image::rbac_min_a.png[,538,align=middle]
++
+The final item required is defining the *Function Scope* under "Eventing / Manage Scope Function" since we will be listing to mutations in a collection under 'bulk.data' it makes sense to use this as our grouping.
```

Comment:
```suggestion
The final item required is defining the *Function Scope* under "Eventing / Manage Scope Function". Since we will be listing to mutations in a collection under 'bulk.data' it makes sense to use this as our grouping.
```